hedera-node/hedera-evm-api/src/main/java/com/hedera/services/evm/contracts/execution/HederaEvmMessageCallProcessor.java

```diff
@@ -36,7 +36,7 @@
 import org.hyperledger.besu.evm.tracing.OperationTracer;
 
 /** Overrides Besu precompiler handling, so we can break model layers in Precompile execution */
-public class HederaEvmMessageCallProcessor extends MessageCallProcessor {
+public abstract class HederaEvmMessageCallProcessor extends MessageCallProcessor {
```

Comment:
We should be able to initialize this class outside of the library (e.g. on mirror-node side), so we should find a way to not make it `abstract`.